commons-numbers-fraction/src/test/java/org/apache/commons/numbers/fraction/BigFractionTest.java

```diff
@@ -778,4 +780,15 @@ void testMath340() {
                                                    fractionA.getDenominator().multiply(fractionB.getDenominator()));
         Assertions.assertEquals(correctResult, errorResult);
     }
+
+    @Test
+    void testNumbers150(){
+        // zero to negative powers should throw an exception
+        assertThrows(ArithmeticException.class, () -> BigFraction.ZERO.pow(-1));
+        assertThrows(ArithmeticException.class, () -> BigFraction.ZERO.pow(Integer.MIN_VALUE));
```

Comment:
We should also have a test for a fraction above 1 raised to the power Integer.MIN_VALUE. This was the original error in numbers-150.

Reply:
@aherbert done.